head_pose_estimator/FaceRect face_rect
geometry_msgs/Quaternion head_rotation
